resources/styles/global/tutor-booksplash-background.less

```diff
@@ -0,0 +1,44 @@
+// Applies a fixed background effect to a container that includes a large fixed header
+//
+// It's intended to be customized for different looks depending on the type of content
+// the user is styling, i.e. Physics, Biology, etc
+.tutor-booksplash-background {
+
+  &:before {
+    content: attr(data-title);
+    position: fixed;
+    left: 0px;
+    right: 0px;
+    overflow: hidden;
+    background-color: @tutor-tertiary;
+    color: @tutor-secondary;
+    text-align: center;
+    font-size: 300px;
+    line-height: 200px;
+    font-family: 'HelveticaNeue-Bold', 'Helvetica Neue Bold', 'Helvetica Neue';
```

Comment:
Yes, I believe it's a proprietary font that's included on OS X only.  I suppose we could license it but is probably better to find an alternative.

@Fredasaurus any thoughts on substituting a different font for HelveticaNeue here?  I'd used it because here because it was in the original mockup.  I'm not sure that was intended or perhaps it was just the default for your mockup tool.   I'd suggest using Lato Bold or we could bundle a different Google font of your choosing.